_config.yml

```
@@ -212,3 +212,10 @@ installation:
       variants:
         - name: "ASUS Tinkerboard"
           key: "tinker"
+
+    generic-x86-64:
+      board: Generic x86-64
+      installation_media: "S-ATA HDD/SSD/M.2 SSD/eMMC module"
```

```suggestion
      installation_media: "storage device"
```
Is there any other possible device? 😄 
How about `"storage device"` as that covers all current and future posibilities?

---

Hehe, well, whatever PC BIOS support, the do support quite a bit. Storage device is fine for me.